testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/mdb/JMSMessageDrivenBeanTestCase.java

```diff
@@ -58,7 +59,8 @@
     public static Archive<?> deployment() {
         final Archive<?> deployment = ShrinkWrap.create(JavaArchive.class, "JMSMessageDrivenBeanTestCase.jar")
                 .addClasses(ReplyingMDB.class,JMSMessagingUtil.class)
-                .addClass(CreateQueueSetupTask.class);
+                .addClass(CreateQueueSetupTask.class)
+                .addClass(TimeoutUtil.class);
```

Comment:
This deployment will now need a security manager permission to read the system property for it to pass in a security-manager enabled run. MDBTestCase has an example of setting that up.